magento/ece-tools
# MAGECLOUD-3598 [Cloud Docker]: Set Varnish as default cache

### Description
Automatically select Varnish as the cache provider when starting Magento from cloud docker.

### Fixed Issues (if relevant)

1. https://github.com/magento/ece-tools/issues/534

### Manual testing scenarios

1. Start Magento cloud docker as described on https://devdocs.magento.com/guides/v2.3/cloud/docker/docker-config.html
2. Verify the current full page caching application by running the following command:
`docker-compose run deploy magento-command config:show system/full_page_cache/caching_application`


### Contribution checklist
